mpp/include/mpp_domains_misc.inc

```diff
@@ -597,6 +605,10 @@ end subroutine init_nonblock_type
                 endif
                 domain%ntiles = msg(12)
              else
+                domain%x(1)%data  %begin = msg(2) - msg(7)
+                domain%x(1)%data  %end   = msg(3) + msg(9)
```

Comment:
should this be `msg(3) + msg(8)`? which i think corresponds to the `ehalo`

Reply:
good catch